Woof/Woof/View/OwnerFlow/OwnerProfileView/OwnerProfileView.swift

```diff
@@ -20,10 +20,8 @@ struct OwnerProfileView: View {
                             address: $viewModel.address
                         )
 
-                        Button(isEditingMode ? saveButtonLabelText : editButtonLabelText) {
-                            if isEditingMode {
-                                viewModel.save()
-                            }
+                        Button(saveButtonLabelText) {
+                            viewModel.save()
                             isEditingMode.toggle()
                         }
```

Comment:
I think this line must be here.
```suggestion
                        }
                        .disabled(isEditingMode && viewModel.name.isEmpty)
```

Reply:
Fixed with Fixed with 53bbc79